src/elements/content-picker/ContentPicker.js

```diff
@@ -290,6 +292,15 @@ class ContentPicker extends Component<Props, State> {
         }
     }
 
+    getSelectedItems = (): BoxItem[] => {
```

Comment:
Can you add comments to document this method, similar to other methods in this file?

Reply:
Yep!